sw/device/lib/testing/kmac/kmac_vector_generate.c

```diff
@@ -0,0 +1,67 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "KeccakHash.h"
+#include "SP800-185.h"
+#include "config.h"
+
+/**
+ * Exit if `condition` is false.
+ *
+ * @param condition Condition to be checked.
+ */
+static void assert(int condition) {
+  if (!condition) {
+    printf("Condition not satisfied.\n");
+    exit(1);
+  }
+}
+
+/**
+ * Print given long byte array with hex characters.
+ *
+ * @param ptr Byte array pointer.
+ * @param len Number of bytes to be printed.
+ */
+void print_long_array(uint8_t *ptr, size_t len) {
+  for (size_t i = 0; i < len; ++i)
+    printf("%02x", ptr[i]);
+  printf("\n");
+}
```

Comment:
Please 
* use `CHECK()` instead of `assert()` (also, `exit()` doesn't make a lot of sense in our environment since we are bare bones and don't have a host environment yet), and
* the `%!x` specifier instead of `print_long_array()`.

```suggestion
```

Reply:
I think my mistake was to put this under `sw/device`, but this is actually host binary. Are these also available for host binaries?

Reply:
Ah, I was confused because of the path. `sw/device` includes only device binaries along with tests (well some are host binaries because our unit tests run on the host). If this is a host helper/tool/utility binary it should go under `sw/host`. But I see that this file is no longer a part of this PR. Thanks @ballifatih, resolving this comment.